README.md

```diff
@@ -2,8 +2,11 @@
 
 A web app for viewing the MobileCoin blockchain.
 
--   Prodiction: https://block-explorer.mobilecoin.foundation
--   Staging: https://block-explorer.stage.mobilecoin.foundation
+-   Prodiction mainnet: https://block-explorer.mobilecoin.foundation
+-   Prodiction testnet:
```

Comment:
Typo: prodiction should be production.  I'm assuming the Missing URLs are in a follow-on PR?

Reply:
Wasn't sure on the URL at the time but just updated to `https://block-explorer.test.mobilecoin.com`